internal/daemon/controller/handlers/hosts/host_service_test.go

```diff
@@ -1232,23 +1287,27 @@ func TestUpdate_Static(t *testing.T) {
 			got, gErr := tested.UpdateHost(auth.DisabledAuthTestContext(iamRepoFn, proj.GetPublicId()), req)
 			if tc.err != nil {
 				require.Error(gErr)
+				assert.Nil(got)
 				assert.True(errors.Is(gErr, tc.err), "UpdateHost(%+v) got error %v, wanted %v", req, gErr, tc.err)
+				if tc.wantErrContains != "" {
+					assert.Contains(gErr.Error(), tc.wantErrContains)
+				}
+				return
 			}
 
-			if tc.err == nil {
-				defer resetHost()
-			}
+			defer resetHost()
+			require.NoError(gErr)
+			require.NotNil(got)
 
-			if got != nil {
-				assert.NotNilf(tc.res, "Expected UpdateHost response to be nil, but was %v", got)
-				gotUpdateTime := got.GetItem().GetUpdatedTime().AsTime()
-				// Verify it is a set updated after it was created
-				// TODO: This is currently failing.
-				assert.True(gotUpdateTime.After(hCreated), "Updated set should have been updated after it's creation. Was updated %v, which is after %v", gotUpdateTime, hCreated)
+			assert.NotNilf(tc.res, "Expected UpdateHost response to be nil, but was %v", got)
+			gotUpdateTime := got.GetItem().GetUpdatedTime().AsTime()
+			// Verify it is a set updated after it was created
+			// TODO: This is currently failing.
+			assert.True(gotUpdateTime.After(hCreated), "Updated set should have been updated after it's creation. Was updated %v, which is after %v", gotUpdateTime, hCreated)
```

Comment:
Sorry, also not you.
```suggestion
			assert.True(gotUpdateTime.After(hCreated), "Updated set should have been updated after its creation. Was updated %v, which is after %v", gotUpdateTime, hCreated)
```